naga/tests/snapshots.rs

```diff
@@ -316,7 +324,20 @@ impl Input {
                 ),
             },
             Err(_) => Parameters::default(),
+        };
+
+        if params.targets.is_none() {
+            match self.input_path().extension().unwrap().to_str().unwrap() {
+                "wgsl" => params.targets = Some(Targets::wgsl_default()),
+                "spvasm" => params.targets = Some(Targets::non_wgsl_default()),
+                "vert" | "frag" | "comp" => params.targets = Some(Targets::non_wgsl_default()),
```

Comment:
style: Why not merge these `match` arms? 🤔

Reply:
Because they're different cases and logically different branches.